python/mqt/qmap/ph/subcircuit_compilation.py

```
@@ -0,0 +1,307 @@
+            optimisation + simulation.
+    """
+
+    performance: dict[str, Any]
+    baseline_performance: dict[str, Any]
+    loss: float
+    baseline_loss: float
+    compute_time: float
+    baseline_compute_time: float
+
+
+def compile_subcircuit(
```

Ruff (0.15.20)

[warning] Too many statements (54 > 50)

(PLR0915)